Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m

```diff
@@ -60,9 +60,6 @@
 [PathVascularMask, bSkipStandard] = xASL_sub_CheckTemplatePath('^MaskVascular', x, bSkipStandard);
 
 [PathT1, bSkipStandard] = xASL_sub_CheckTemplatePath('^T1', x, bSkipStandard);
-[PathpGM, bSkipStandard] = xASL_sub_CheckTemplatePath('^pGM', x, bSkipStandard);

```

Comment:
Why are we using MNI space masks instead of using the individual masks? That defies the use of group mask completely, right?

I found the original version you wrote much better, we only had to change pGM to PV_pGM...